tests/test_stanford_publication_citations/test_stanford_publication_citations.info.yml

```diff
@@ -0,0 +1,4 @@
+name: 'Test Stanford Publications Citations'
+type: module
+core_version_requirement: ^8.8 || ^9
+hidden: true
```

Comment:
Ooof, this has a lot of duplication from the core set of config. Anything you've got to ensure that changes to these fields get copied into the test module?

Reply:
Probably when we set up a UI for field mapping. But as long as the tests pass, it's actually a good sign

Reply:
Can you add something to automatically sync the configs from config to here on PR or something to keep these intact?

Reply:
That's going to be tricky because this doesn't include everything from regular install. Would that be a drush command or something else.... :shrug: I think cherry picking the configs manually is better for this scenario IMO